src/Serilog.Sinks.AzureTableStorage/Sinks/KeyGenerator/DefaultKeyGenerator.cs

```diff
@@ -0,0 +1,27 @@
+using System.Threading;
+using Serilog.Events;
+
+namespace Serilog.Sinks.AzureTableStorage.KeyGenerator
+{
+    class DefaultKeyGenerator : IKeyGenerator
+    {
+        protected long RowId;
+
+        public DefaultKeyGenerator()
+        {
+            RowId = 0L;
+        }
+
+        public string GeneratePartitionKey(LogEvent logEvent)
+        {
+            var utcEventTime = logEvent.Timestamp.UtcDateTime;
+            var timeRoundedToMinute = utcEventTime.AddMilliseconds(-utcEventTime.Millisecond);
+            return $"0{timeRoundedToMinute.Ticks}";
+        }
+
+        public string GenerateRowKey(LogEvent logEvent)
+        {
+            return $"{Interlocked.Increment(ref RowId)}";
```

Comment:
I think it even was a combination of those two and a row ID. The problem with that is that it reorders the messages in the log. My idea is that if you open up the table you see the messages in the order they were written. (or reverse order as we now have in production)